tests/transactional/verify_undelete_snapshots.pm

```diff
@@ -35,7 +35,7 @@ sub run {
                                 creates snapshots and installs a package";
     assert_script_run("snapper create -d \"Disposable snapshot #1\"");
     get_utt_packages;
-    trup_call "ptf install" . rpmver('security');
+    trup_call "ptf install" . rpmver('trivial');
```

Comment:
Hmm, are we sure that we can simply replace the package here? Is it clear what is the problem when `security` is used?

Reply:
is already installed in the other children job. I insisted a time ago to avoid mixing up in this scenario 'trup_calls' with snapper commands, so basically here could be anything for the purpose of the test, I've just changed it to one not conflicting.

Reply:
Ah, I see now. Removing the package would introduce other issues, so it's indeed easier to use different package here. Thanks for the explanation!